app/views/global_user_roles/edit.html.haml

```diff
@@ -0,0 +1,27 @@
+= content_for :h1 do
```

Comment:
I think we could, though one hitch is we only allow one facility role per user, so the role `<select>` would be different (maybe *very* different if we go with checkboxes for the global UI).